usagov/vote-gov
# VOTE-412 -> add cypress test to check footer on page

link to PR -> https://bixal-projects.atlassian.net/browse/VOTE-412

Purpose of PR -> with the addition of a new footer, this cypress test will check that each link in the footer will take the user to the correct and expected page after being clicked on.  Can also check the logs for the Cypress Function testing suite under the Github actions. 

### How to test

- pull down the branch 
- run `npm run start`
- spilt terminal 
- run `npm run cy:open` this will open the cypress runner 
- select the `footer.cy.js` test and verify that test is passing
